pkg/sbom/scanner/scanner.go

```diff
@@ -81,8 +81,8 @@ func NewScanner(cfg config.Component, collectors map[string]collectors.Collector
 // CreateGlobalScanner creates a SBOM scanner, sets it as the default
 // global one, and returns it. Start() needs to be called before any data
 // collection happens.
-func CreateGlobalScanner(cfg config.Component, wmeta option.Option[workloadmeta.Component]) (*Scanner, error) {
-	if !cfg.GetBool("sbom.host.enabled") && !cfg.GetBool("sbom.container_image.enabled") && !cfg.GetBool("runtime_security_config.sbom.enabled") {
+func CreateGlobalScanner(coreConfig config.Component, systemProbeConfig config.Component, wmeta option.Option[workloadmeta.Component]) (*Scanner, error) {
+	if !coreConfig.GetBool("sbom.host.enabled") && !coreConfig.GetBool("sbom.container_image.enabled") && !systemProbeConfig.GetBool("runtime_security_config.sbom.enabled") {
```

Comment:
when called from pkg/security/resolvers/sbom/resolver.go, we use systemprobe for all 3 settings. Is sbom in datadog.yaml of system-probe.yaml ?

Reply:
On the PR I changed to use core config and systemprobe config 

https://github.com/DataDog/datadog-agent/blob/2a2b47980a92fcb3b58beb721b9abfd5a50b740c/pkg/security/resolvers/sbom/resolver.go#L147

`"sbom.host.enabled"` -> Is defined on both 
https://github.com/DataDog/datadog-agent/blob/ed30a47fc978ab043632312d7d2f5dd8e68dfcce/pkg/config/setup/config.go#L879

https://github.com/DataDog/datadog-agent/blob/fc6656117afa235089f7e4acea9d7048c6d0c610/pkg/config/setup/system_probe.go#L78

`"sbom.container_image.enabled"` -> Is defined in the core config
https://github.com/DataDog/datadog-agent/blob/ed30a47fc978ab043632312d7d2f5dd8e68dfcce/pkg/config/setup/config.go#L869